### ISSUE TITLE
v1.3.0 not available for install

### ISSUE THREAD

**Issue by rolinger** [sarriaroman/photoviewer#230]
trying to install v1.3.0:

`ionic cordova plugin add com-sarriaroman-photoviewer@1.3.0` and the fetch is failing.....

Then I saw this in a closed issue:  https://github.com/sarriaroman/photoviewer/issues/228

With a reference to installing it via GIT, but I can't find the reference to the GIT anywhere mentioned in issue #228 - can someone please post it here or update the documentation.

**Comment by rolinger** [comment]
@sarriaroman - please make 1.3.0 public

**Comment by sarriaroman** [comment]
Ups... really sorry, I will publish it today. Thanks for letting me know

**Comment by rolinger** [comment]
@sarriaroman  Thank you....this will also make the fixes from 1.2.5 public too - both 1.2.5 and 1.3.0 were not public, installing @1.2.5 or @latest was still installing 1.2.4.

Please let us know when its made public.  As of 11:20am (EST) its still not available for install.

**Comment by rolinger** [comment]
@sarriaroman - any update on this?  v1.3.0 still isn't available for install.

**Comment by sithwarrior** [comment]
@rolinger to install it directly from git, you call do it like this:

`Cordova plugin add https://github.com/sarriaroman/photoviewer`

This will work for now. However when the project updates the master branch, so will the plugin you have installed, if you reinstall etc.

**Comment by rolinger** [comment]
@sithwarrior - have you actually tried that?  Because it fails for me.  Can't find the git binary.

> cordova.cmd plugin add com-sarriaroman-photoviewer@1.3.0
```
Failed to fetch plugin com-sarriaroman-photoviewer@1.3.0 via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
CordovaError: Error: npm: Command failed with exit code 1 Error output:
npm ERR! code ETARGET
npm ERR! notarget No matching version found for com-sarriaroman-photoviewer@1.3.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\rolin\AppData\Roaming\npm-cache\_logs\2023-03-14T13_49_06_974Z-debug.log
[ERROR] An error occurred while running subprocess cordova.

        cordova.cmd plugin add com-sarriaroman-photoviewer@1.3.0 exited with exit code 1.

        Re-running this command with the --verbose flag may provide more information.
```

> cordova.cmd plugin add https://github.com/sarriaroman/photoviewer
```
Failed to fetch plugin https://github.com/sarriaroman/photoviewer via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
CordovaError: Error: No git binary found in $PATH
[ERROR] An error occurred while running subprocess cordova.

        cordova.cmd plugin add https://github.com/sarriaroman/photoviewer exited with exit code 1.

        Re-running this command with the --verbose flag may provide more information.
```

**Comment by sithwarrior** [comment]
Yes I have tried that, that's how I have it installed at the moment.

Maybe your Cordova version is outdated or something? Or perhaps you don't have GIT installed 

`Error: No git binary found in $PATH`

**Comment by rolinger** [comment]
@sithwarrior - I guess I don't understand what you are saying.

These install:
```
cordova plugin add com-sarriaroman.photoviewer (v1.2.4)
cordova plugin add com-sarriaroman.photoviewer@latest (v1.2.4)
cordova plugin add com-sarriaroman.photoviewer@1.2.4
cordova plugin add com-sarriaroman.photoviewer@1.2.5 (though, this still installs 1.2.4)
```

But these don't install, (git path not found):
```
cordova plugin add com-sarriaroman.photoviewer@1.3.0
cordova plugin add https://github.com/sarriaroman/photoviewer
cordova plugin add https://github.com/sarriaroman/photoviewer.git
cordova plugin add https://github.com/sarriaroman/photoviewer#master
```

**Comment by sithwarrior** [comment]
`git path not found`

is probably because you need to have git, the source control installed, before Cordova can download the plugin directly from GitHub.

[https://git-scm.com/downloads](https://git-scm.com/downloadsl)

When you use the first methods, with the plugin name, Cordova is using NPM to fetch the package.

**Comment by rolinger** [comment]
Ok...so small snafu on my part.  I had installed git on a previous computer and not my new one - though I thought I had.   That being said, if @sarriaroman would fully publish or make public the newest version, then NPM standard install would work.   The newest version has been merged for over 4 months, but its still not setup correctly for standard install.

However, thank @sithwarrior - your feedback got this resolved for me.  My mistake was not realizing that git wasn't installed as (in my mind) I thought it was and thought I had installed other packages using native git - but I realize now I did install other packages using git; but just not on THIS computer.
